src/app.ts

```diff
@@ -43,15 +49,15 @@ const showMatches = async (
         if (line.match(new RegExp(`.*${match}.*`))) {
           const columnNumber = line.search(match) + 1;
 
-          if (args['luhn-check'] && isCardNumber(match)) {
+          if (args['luhn-check'] && isCardNumber(match, args['exclude-numbers'])) {
```

Comment:
You should be able to reference these args as `args.ignoreNumbers` and `args.luhnCheck`. Yargs should automatically convert them to camel case.